samples/scala-valueentity-counter/build.sbt

```diff
@@ -40,14 +40,9 @@ val logback = "ch.qos.logback" % "logback-classic" % LogbackVersion
 val logbackJson = "ch.qos.logback.contrib" % "logback-json-classic" % LogbackContribVersion
 val logbackJackson = "ch.qos.logback.contrib" % "logback-jackson" % LogbackContribVersion
 
-// FIXME include these dependencies via the AkkaserverlessPlugin
-val AkkaServerlessSdkVersion = System.getProperty("akkaserverless-sdk.version", "0.7.2")
 libraryDependencies ++= Seq(
   logback,
   logbackJson,
   logbackJackson,
```

Comment:
Still some logback dependencies here that shouldn't be needed. They should be included via `akkaserverless-scala-sdk` unless we changed that.